modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm

```diff
@@ -131,10 +131,8 @@ sub get_pubmed_prefix {
 
 =head2 core_db_adaptor
 
-  Arg [1]    : Bio::EnsEMBL::DBSQL::DBAdaptor $db_adaptor (optional)
```

Comment:
I know I'm being neurotic here, however the (optional) parameter is no longer relevant and should be removed

Reply:
I agree, this line is already removed.